compiler/rustc_data_structures/src/base_n/tests.rs

```diff
@@ -1,6 +1,7 @@
 use super::*;
 
 #[test]
+#[cfg_attr(miri, ignore)] // this takes way too long under miri
```

Comment:
Rather than ignoring the test entirely, would it make sense to change the `1000` below to `10` when running in Miri?

Reply:
Oops, I completely overlooked that inner for loop. Yes, changing it to 10 makes it fast enough.